wordpress-mobile/WordPress-iOS
# Onboarding Improvements: Add skip button to Epilogue if there are no sites yet

Fixes #17506 

## Description
This PR adds a skip button to the Epilogue screen if there are no sites yet 

Slack ref: p1637330082057200/1637317113.044600-slack-C027K4MNPGQ

Before | After
-- | --
<img src="https://user-images.githubusercontent.com/6711616/142911973-dd3232bd-7ace-4198-8e4a-a25996539883.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911540-eafa6c2f-7d4f-4732-9cf3-5c775b4bcea5.png" width=200>

## How to test

1. Log out / log in to a site w no sites
2. Make sure you see the Post Signup Interstitial screen (see below for an example)

<img src="https://user-images.githubusercontent.com/6711616/142620725-b2153bdd-0c3b-44d8-8744-f7ac1ebea5f7.PNG" width=200>

3. ✅ The skip button should be showing
4. Tap the skip button
5. ✅ The Epilogue should be dismissed

Case | JP iPhone | WP iPhone | WP iPad
-- | -- | -- | --
No sites |  <img src="https://user-images.githubusercontent.com/6711616/142914263-27d5e4dd-6535-496f-97e4-d0ece9e57e6f.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911540-eafa6c2f-7d4f-4732-9cf3-5c775b4bcea5.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911681-604f8152-ba37-40bc-b437-72f1f465710f.png" width=200> 
1-3 sites | <img src="https://user-images.githubusercontent.com/6711616/142914261-4b81fd12-3d86-400a-9f70-252aab7c5ae4.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911544-d4339ab1-3aa7-4200-8301-d3506e1abb78.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911686-5ed325fc-e8f7-41c2-a811-4efa193a6bbc.png" width=200> 
4+ sites |  <img src="https://user-images.githubusercontent.com/6711616/142914254-2e822db0-ce83-47bf-b5ce-a0e914d77115.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911552-f155ffea-deb5-4f9c-8346-d2fcc6efdf56.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142911690-c3502165-ea30-40b6-9122-2cc0e4a3e7e5.png" width=200> 


## Regression Notes
1. Potential unintended areas of impact
- iPad readable content width
- Epilogue for 1-3 sites
- Epilogue for 4+ sites
- Jetpack app (Create a site should NOT be showing in the Epilogue ever)

2. What I did to test those areas of impact (or what existing automated tests I relied on)
- Tested the above steps manually

3. What automated tests I added (or what prevented me from doing so)
- N/A

PR submission checklist:
